app/_hub/kong-inc/file-log/index.md

```diff
@@ -79,14 +79,14 @@ params:
 
 ---
 
-## Log Format
+## Log format
 
 {% include /md/plugins-hub/log-format.md %}
 
 ### JSON object considerations
 
 {% include /md/plugins-hub/json-object-log.md %}
 
-## Kong Process Errors
+## Kong process errors
```

Comment:
Since this could be either enterprise or oss, and variables don't work in titles, would rather just leave as is for Kong process errors